larreco/QuadVtx/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+
+art_make(BASENAME_ONLY
+         LIB_LIBRARIES
+           larcorealg_Geometry
+           ${ART_FRAMEWORK_SERVICES_REGISTRY}
+           ${ROOT_CORE}
+           ${ROOT_HIST}
+           ${ROOT_PHYSICS}
+           canvas
+           cetlib_except
+         MODULE_LIBRARIES
+           larcorealg_Geometry
+           lardataobj_RecoBase
+           larreco_QuadVtx
+           nusimdata_SimulationBase
+           lardata_ArtDataHelper
+           ${ART_ROOT_IO_TFILESERVICE_SERVICE}
+           ${ART_ROOT_IO_TFILEDIRECTORY}
+           ${ART_ROOT_IO_TFILE_SUPPORT}
+           ${ART_FRAMEWORK_SERVICES_REGISTRY}
+           ${ROOT_BASIC_LIB_LIST}
```

Comment:
The `${ROOT_BASIC_LIB_LIST}` is quite large.  Do you really need it all?

Reply:
which variable will give me the libraries for TVectorT et al?

Reply:
I believe `${ROOT_HIST}` is sufficient for `TVectorT`.

Reply:
Apparently not, but I fiddled some more and adding Matrix, Physics, and Tree has it building.